tutor/commands/config.py

```diff
@@ -230,8 +234,31 @@ def patches_list(context: Context) -> None:
     renderer.print_patches_locations()
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = tutor_config.config_path(context.root)
+
+    if not os.path.isfile(config_file):
+        raise exceptions.TutorError(f"Missing config file at {config_file}")
+
+    open_cmd = None
+
+    if which("open"):  # MacOS & linux distributions that ship `open`. eg., Ubuntu
+        open_cmd = ["open", config_file]
+    elif which("xdg-open"):  # Linux
+        open_cmd = ["xdg-open", config_file]
+    elif which("start"):  # Windows
+        open_cmd = ["start", '""', config_file]
+    else:
+        raise exceptions.TutorError(f"Failed to find utility to launch an editor.")
+
+    subprocess.call(open_cmd)
+
```

Comment:
What do you think about adding a message at the end here along the lines of `Remember to run 'tutor config save' in order to apply configuration edits to your environment.`